src/main/kotlin/mate/academy/model/IsoscelesTrapezoid.kt

```diff
@@ -0,0 +1,20 @@
+package mate.academy.model
+
+class IsoscelesTrapezoid(color: Color,
+                         private val base1: Double,
+                         private val base2: Double,
+                         private val height: Double) : Figure(color) {
+
+    companion object {
+        private const val HALF = 0.5
```

Comment:
This is a little overengineering as this value is used only inside the IsoscelesTrapezoid

Reply:
yeah, but if I don’t make a constant, then checkstyle swears(

Reply:
Bloody hell :) 
But try to avoid such usage in real projects